src/Storage/Device/OracleObject.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Utopia\Storage\Device;
+
+use Exception;
+use Utopia\Storage\Device;
+use Utopia\Storage\Storage;
+
+class OracleObject extends Device
```

Comment:
This should probably extend the [S3](https://github.com/utopia-php/storage/blob/main/src/Storage/Device/S3.php) adapter because Oracle Object is an S3-compatible storage provider.